src/_scss/components/_moreOptionsTabs.scss

```diff
@@ -0,0 +1,156 @@
+.more-options__tabs_primary {
+    @import "pages/search/results/table/_tableTypes";
+    @include display(flex);
+
+    .tab-padding-right {
+        padding-right: rem(15);
+        border-bottom: 1px solid $color-gray-light;
+    }
+
+    .table-type-toggle {
+        white-space: nowrap;
+    }
+
+    .usa-dt-picker {
+        @include display(flex);
+        justify-content: center;
+        width: 100%;
+        @include media($tablet-screen) {
+            flex: 1 1 0%;
+        }
+        height: rem(50);
+        background-color: $color-gray-lightest;
+        border-bottom: 1px solid $color-gray-light;
+
+        &:hover{
+            @include transition(all 0.3s ease-in-out);
+            background-color: darken($color-gray-lightest, 5%);
+        }
+        &.active {
+            background-color: $color-white;
+            border-top: 1px solid $color-gray-light;
+            border-right: 1px solid $color-gray-light;
+            border-bottom: 1px solid $color-white;
+            border-left: 1px solid $color-gray-light;
+            color: $color-base;
+
+            &:hover{
+                @include transition(all 0.3s ease-in-out);
+                background-color: darken($color-gray-lightest, 5%);
+            }
+        }
+
+
+        .usa-dt-picker__dropdown-container {
+            @include display(flex);
+            width: 100%;
+            height: 100%;
+            z-index: 999;
```

Comment:
Let's see if we can find the minimum z-index value (we've tried to stick to low values like 2, 3, 4)

Reply:
Can I update this in my other PR with the table?